docusaurus/docs-internals/editor-controls/group-controls/popover.md

```diff
@@ -0,0 +1,263 @@
+| `type`                 | `string`                                                                                                                                                                                                                                |     -      | Type should be `"popover"` to use this control                                                                                                                                                                                   |
+| `position?`            | `number`                                                                                                                                                                                                                                |     -      | The position of the control in toolbar                                                                                                                                                                                           |
+| `devices?`             | `"all"` \| `"desktop"` \| `"responsive"`                                                                                                                                                                                                |  `"all"`   | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices |
+| `disabled?`            | `boolean`                                                                                                                                                                                                                               |  `false`   | Configure the condition under which the control is disabled or enabled                                                                                                                                                           |
+| `options?`             | `Array<ControlItem>`                                                                                                                                                                                                                    |     -      | An array of controls that will be displayed inside the popover <br/><br/> **`ControlItem = { id: number; type: string; label?: string; disabled?: boolean; position?: number; devices?: "all" \| "desktop" \| "responsive" }`**  |
```

Comment:
need to explain every keys of the `ControlItem` in options description, example here: https://github.com/EasyBrizy/Brizy-Local-Editor/pull/284/files#diff-50fa469c71578c788103f8a321485b845804a91d4cf428fd95eec9a7312bbc46R21

Reply:
maybe better introduce some global concept of `Control` to not duplicate all the text about id, label, type...
because `Control` can have also a helper, config, other `options` with the other `Control` (`popover` - `group` - items`) and it can be too much to describe in the docs

Reply:
better from what point of view ?
better because info is not duplicated ?

we do as we do now not to go down the path to other documentation, when you want to read something you must have
1kk opened tabs